### ISSUE TITLE
How to train my own dataset?

### ISSUE THREAD

**Issue by EddieLv** [marcovarrone/cellcharter#4]
### Description of feature

Thanks for your excellent work!
Now I did not see how to train my own dataset in your tutorial, I cannot wait to run the full pipeline:)

**Comment by marcovarrone** [comment]
Thank you @EddieLv!

I am writing the tutorials on both running CellCharter on spatial transcriptomic data and on the full pipeline of spatial proteomics.
Which type of data are you interested in? So I can prioritize one or the other :)

**Comment by EddieLv** [comment]
Thanks for replying! I am interested in ST data, I cannot wait to see the tutorial :)

**Comment by dpcook** [comment]
Just bumping this to express interest in some extended tutorials and/or API details to reproduce ST workflows similar to those demonstrated in the preprint. Really looking forward to trying CellCharter out!

**Comment by marcovarrone** [comment]
Sorry for taking long but I have been quite busy lately.
I uploaded a [notebook](https://cellcharter.readthedocs.io/en/latest/notebooks/cosmx_human_nsclc.html) showing a use case of CellCharter applied to the Non-Small Cell Lung Cancer CosMx dataset from Nanostring

For 10x Visium the procedure is the same.

I suggest updating CellCharter with `pip install cellcharter --upgrade` because I made some small changes to the code.

Let me know if you encounter any problem :) I would be happy to help.
